src/PoolAgent.js

```diff
@@ -37,6 +37,30 @@ class PoolAgent extends Nimiq.Observable {
         /** @type {Nimiq.Timers} */
         this._timers = new Nimiq.Timers();
         this._timers.resetTimeout('connection-timeout', () => this._onError(), this._pool.config.connectionTimeout);
+
+        // Public interface
+        Object.defineProperties(this, {
```

Comment:
We use [ES6 getters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get) in our codebase instead.